### PR TITLE
Configure missing area around the world

### PR DESCRIPTION
**Pull request by frodrigo**
Add non configured area.

All this area should also be configured in download.osm.fr

### CODE REVIEW

File: osmose_config.py [hunk 1]
```diff
@@ -542,6 +542,7 @@ class gen(default_country):
     phone_code="687", phone_len=6, phone_format=r"^[+]%s([- ./]*[0-9]){5}[0-9]$", phone_international='00')
 
 default_country("merge", "france_taaf", 2186658, download_repo=OSMFR, analyser_options={"country": "TF", "language": "fr", "proj": 32738})
+default_country("merge", "clipperton", 2573009, download_repo=OSMFR, analyser_options={"country": "CP", "language": "fr", "proj": 32612})
```
Comment (jocelynj) [inline osmose_config.py:545]: I'm creating it in oceania/clipperton

File: osmose_config.py [hunk 2]
```diff
@@ -1128,9 +1131,11 @@ class gen(default_country):
 south_africa_province("north_west", 349519)
 south_africa_province("northern_cape", 86720)
 south_africa_province("western_cape", 80501)
+south_africa_province("prince_edward_islands", 52947809)
```
Comment (jocelynj) [inline osmose_config.py:1134]: This relation doesn't exist in OSM.

**Comment by jocelynj** [comment]
All polygons, except for prince_edward_islands, are now generated on download.openstreetmap.fr

File: osmose_config.py (hunk 2 above)
Comment (frodrigo) [reply osmose_config.py:1134]: Hum, it is a way id. There is no relation.

**Comment by jocelynj** [comment]
Could you update this PR by:
  - removing prince_edward_islands
  - moving clipperton to oceania ?

Then, I will be able to merge it to github, and add the missing regions to buildbot.

**Comment by frodrigo** [comment]
> Could you update this PR by:
> 
>     * removing prince_edward_islands
> 
>     * moving clipperton to oceania ?
> 
> 
> Then, I will be able to merge it to github, and add the missing regions to buildbot.

Done.

Note, the Clipperton extract is still missing on OSM-FR.

**Comment by jocelynj** [comment]
> Note, the Clipperton extract is still missing on OSM-FR.

It is now present on http://download.openstreetmap.fr/extracts/oceania/.

**Pull request merged**